moduleroot/Gemfile

```diff
@@ -25,7 +27,8 @@ def location_for(place_or_version, fake_version = nil)
   end
 end
 
-<% if @configs['supports_windows'] -%>
+<%# Workaround to support boolean and array supports_windows settings until one form is no longer used %>
+<% if @configs['supports_windows'] == true or (@configs['supports_windows'].is_a?(Array) and @configs['supports_windows'].include? @configs[:puppet_module]) -%>
```

Comment:
Can we not use and/or logic here? Prefer `&&/||` for boolean expressions and `and/or` for control flow.